app/main/views/users.py

```diff
@@ -113,6 +113,7 @@ def create_user():
     now = datetime.utcnow()
     user = User(
         email_address=json_payload['emailAddress'].lower(),
+        phone_number=json_payload.get('phoneNumber'),
```

Comment:
Should we do `phone_number=json_payload.get('phoneNumber') or None` here, so that we don't store empty strings as well as NULLs?